app/helper_functions.py

```diff
@@ -0,0 +1,29 @@
+from flask import make_response, abort
+from dotenv import load_dotenv
+import os
+import requests
+
+load_dotenv()
```

Comment:
You should be able to access values from `os` without importing `load_dotenv` so you can delete line 2 and 6. load_dotenv() was invoked during start up in create_app().